### DISCUSSION TITLE
Retain STAT/xx/MEM and STAT/xx/RULE MQTT message

### DISCUSSION THREAD

**Discussion by cristianrinaldini** [arendst/Tasmota#15184] [Q&A] [answered]
Is it possible to set retention fro MQTT message related to STAT?
Like it is possible for for STATUS?

Thanks

**Comment by sfromis** [comment, marked as answer]
No. What I do is to have my backend save what I like to have saved, one possible tool to make that easy is Node-Red.

**Reply by barbudor** [reply]
Those messages doesn't make much sense to be retained
But you can always have rules to use Publish2 to publish your mem values to a retained topic
